docs/SUMMARY.md

```diff
@@ -6,7 +6,7 @@
     * [Prerequisites](usage.md#prerequisites)
     * [Docker environment discovery](usage.md#docker-environment-discovery)
     * [Usage modes](usage.md#usage-modes)
-    * [Maven dependencies](usage.md#maven-dependencies)
+    * [Gradle/Maven dependencies](usage.md#gradle-maven-dependencies)
```

Comment:
Will this link still work? So the / becomes a -?

Reply:
You're right. This should be `gradlemaven-dependencies`. I'll fix that.